marlowe/src/Language/Marlowe/Pretty.hs

```diff
@@ -0,0 +1,79 @@
+{-# LANGUAGE DefaultSignatures #-}
+{-# LANGUAGE FlexibleContexts  #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE TypeOperators     #-}
+module Language.Marlowe.Pretty (prettify, Pretty, pretty) where
+
+import           GHC.Generics            ((:*:) ((:*:)), (:+:) (L1, R1), C, Constructor, D, Generic, K1 (K1), M1 (M1),
+                                          Rep, S, U1, conName, from)
+import           Text.PrettyPrint.Leijen (Doc, comma, encloseSep, hang, lbracket, line, parens, rbracket, space, text)
+
+appendWithLine :: Doc -> Doc -> Doc
+appendWithLine x y = x <> line <> y
+
+-- | This function will pretty print an a but will not wrap the whole
+-- expression in parentheses, where as @pretty@ will.
+--
+-- >>> pretty $ MyData One (MyData One Two)
+-- (MyData One (MyData One Two))
+--
+-- >>> prettify $ MyData One (MyData One Two)
```

Comment:
Is the `ify` bit a hangover from a previous version?

Reply:
just this naming thing we talked about yesterday, happy to change if you think the `prettyFragment` is better?

Reply:
ok changed to `prettyFragment`